ui/src/app/components/apps-panel.tsx

```diff
@@ -0,0 +1,3 @@
+import * as React from 'react';
+
+export const AppsPanel = () => <div>TODO</div>;
```

Comment:
The real AppsPanel will look more like #7147 - but it needs some rework.

This will only be visible in environments where thee Feature Flag is true.